kubespawner/spawner.py

```diff
@@ -274,6 +291,32 @@ def _hub_connect_port_default(self):
         """
     )
 
+    storage = Unicode(
+        "1Gi",
+        config=True,
+        help="""
+        The ammount of storage space to request from the volume that the pvc will
+        mount to.
+        """
+    )
+
+    storage_class = Unicode(
+        "",
+        config=True,
+        help="""
+        The storage class that the pvc will use. If left blank, the kubespawner will not
+        create a pvc for the pod.
+        """
+    )
+
+    access_modes = List(
```

Comment:
user_storage_access_modes? Also perhaps use a sane default here (RWX?) so that people don't have to touch it if they don't have to? 

In general, either properties should be required, in which case set their default to None and refuse to work if they aren't set, or be optional and have sane defaults. So this one looks like it should have sane defaults.